### ISSUE TITLE
Preserve undo history after clearing properties tab

### ISSUE THREAD

**Issue by valtism** [fwouts/previewjs#920]
### Describe the bug

Hello again 👋🏻 (can you tell I'm finally using the extension properly 😂? It's super great btw).

If you clear the Properties input using the refresh icon, you cannot get back what you have with cmd + z. It would be great to preserve history to undo a clear done by mistake.

[aside]
I have hit this a few times now accidentally thinking it would refresh the page. I would consider changing the icon to something like a 🚫 or a 🗑

### Reproduction

1. Open the Properties pane in text mode
2. Type some properties in
3. Clear with clear button
4. Try to undo with cmd + z

### Preview.js version

v1.11.0

### Framework

React 17

### System Info

```shell
MacOS 12.5
VS Code 1.70.1
```

### Used Package Manager

yarn

**Comment by fwouts** [comment]
Good idea! Would you imagine that working with the structured props editor too?

**Comment by valtism** [comment]
Yes, that would be ideal!

**Comment by valtism** [comment]
Also, is it a known bug that functions don't really work in the GUI props editor? I can't really do much with them there and have to use the code view to input them. I can open a new issue if not

**Comment by fwouts** [comment]
At the moment function props can only return a value, they can't have a body. Probably worth opening another issue for this explaining your use case for it!

**Comment by valtism** [comment]
Will do, thanks!
